next/src/custom/order.ts

```diff
@@ -1,19 +1,14 @@
 import type { Field } from '../field/type'
 import type { JsfSchema } from '../types'
 
```

Comment:
scout: Can you update `mockGroupArrayInput` to look better with JSF v1 in the Playground? We need to replace `'x-jsf-presentation'.options` with `oneOf`? Otherwise the radio options render without a title/label

cc @dragidavid

Reply:
@lukad I think you forgot this one 😢

Reply:
Ah yes, sorry. Is there something we need to do int jsf for this?

Reply:
@dragidavid just pushed 305d7cf803789d3792783fe1eedf67118d081558 to this branch updating mockGroupArrayInput as mentioned in the other thread. So resolving this one.